src/components/Sections/SectionDate.js

```diff
@@ -7,7 +7,7 @@ import { DEFAULT_DATE_TIME_FORMAT, PARSING_STRING_WITH_TIME_ZONE } from '../../c
 
 export function dateToMoment(date) {
   if (!date) {
-    return moment();
+    return ' N/A';
```

Comment:
i wouldn't expect `N/A` to have a space since the spacing should be the same with a value. Any reason there is a space here?

Reply:
just looks better, otherwise will get `Date:N/A` instead `Date: N/A`

Reply:
fixed. all tets passed